transformers/uws/server.js

```diff
@@ -69,6 +69,8 @@ module.exports = function server() {
     spark.emit('incoming::data', msg);
   });
 
+  native.server.group.onPing(group, (msg, spark) => spark.heartbeat());
```

Comment:
One last request. I noticed that in `ws` this was changed to https://github.com/primus/primus/blob/fb18afa00c20aac45666e80aa8f4c8f216583d9a/transformers/websockets/server.js#L54-L56. Note the `incoming::ping` event.

If I remember correctly this was for a race condition that could crash the server. I think the same should be done for uws (emit `incoming::ping` instead of calling `spark.heartbeat()`).

I'll try to find the relevant issue.